### ISSUE TITLE
[Suggestion] Lower griefing rules

### ISSUE THREAD

**Issue by FutureCW** [EarthMC/Issue-Tracker#500]
_imdoingthisforissue500_

Shouldn't have to be warned for breaking >10 blocks during a battle and not replace them. It's pretty damn obvious that unless it's a major grief, the player who reported it is a dick who wants to get people warned. Being more relaxed on these rules would stop feeding the dicks who report just to get enemies warned.

**Comment by Pengun78** [comment]
I once got warned for grief bc i covered a little water pool with dirt which would later be covered anyway😎😎😎😎

**Comment by Kyoris** [comment]
I mean I would like having grief warns, but not to the pettiness of a few blocks... If the thing is clearly made by a person, say just a few chunks out of a town down a road, it should be warnable, but if it's just near a town (and not along any sort of civilization) and/or very minor to the point of just a few blocks I'm fine with replacing it myself. (though still, fucking newfags need to stop stealing the glowstone from my lamp posts ffs)

**Comment by Pengun78** [comment]
Lol when i make a house and im gonna claim thst chunk the next day, all the red bricks are stolen out of it, same for simple city walls, for that people should get a grief warning yes

**Comment by FutureCW** [comment]
But for when someone breaks a few blocks during a fight near a town, you shouldn't be warned for grief

**Comment by watergod1234** [comment]
I agree! I once was warned for making a few **grass paths**. Yep, that's right.

**Comment by BowShot118** [comment]
I've been threatened with a ban and warns by Sgt and John for placing 3 blocks outside of a town before, alongside just standing in claims.

It was great

**Comment by Ultil0l** [comment]
Battle structures shouldnt be grief too,
they can be easily rolledback

**Comment by Kyoris** [comment]
so basically:
-no petty warns (<10-20 blocks)
-no warns for obvious attempts to help with something/minor accidents (eg. what happened to pengun and watergod)
-no warns for battle structures
-rollbacks are always available for the above if needed

**Comment by FutureCW** [comment]
So rollback minuscule griefs/battle structures

**Comment by CloakedEagle** [comment]
Personally, I do not warn players if they had made a few blocks outside of the claim during a fight. Then again, not every case are the same, hence we make the decision depending on the case and the situation.

If you believe your punishment is not justified, you should speak to the mod/admin who warned you. If you be reasonable, I'm sure they will listen to you.

I would still suggest clearing out the blocks you had placed during a fight, near a town claim.
